gmocoin/private/dto.py

```diff
@@ -9,14 +9,14 @@
 
 from ..common.dto import BaseSchema, BaseResponse, BaseResponseSchema, \
     Symbol, AssetSymbol, SalesSide, OrderType, ExecutionType, SettleType, \
-    OrderStatus, TimeInForce
+    OrderStatus, TimeInForce, MarginCallStatus
 
 
 class GetMarginData:
     """
     余力情報データクラスです。
     """
-    def __init__(self, actual_profit_loss: Decimal, available_amount: Decimal, margin: Decimal, profit_loss: Decimal) -> None:
+    def __init__(self, actual_profit_loss: Decimal, available_amount: Decimal, margin: Decimal, margin_call_status: MarginCallStatus, margin_ratio: Decimal, profit_loss: Decimal) -> None:
```

Comment:
7/18段階でテストを実行すると、`margin_ratio`が返却されていないため、エラーとなりますので、  
返却されない場合デフォルト値-1としていただきたいです。

```suggestion
    def __init__(self, actual_profit_loss: Decimal, available_amount: Decimal, margin: Decimal, margin_call_status: MarginCallStatus, profit_loss: Decimal, margin_ratio: Decimal = -1) -> None:
```

Reply:
ご確認ありがとうございます。Commitさせていただきました。